cram_3d_world/cram_bullet_reasoning/src/debug-window.lisp

```diff
@@ -132,6 +147,87 @@
         (push *current-costmap-sample* (gl-objects *debug-window*))))))
 
 
+
+
+(defun add-vis-axis-object (object &optional (length 0.30))
+  "spawn axis on the object that is given with 3 rigid bodies type box"
+  (sb-thread:with-mutex (*debug-window-lock*)
+    (when (and *vis-axis-x* *debug-window*)
+      (setf (gl-objects *debug-window*)
+            (remove *vis-axis-x* (gl-objects *debug-window*)))
+      (setf (gl-objects *debug-window*)
+            (remove *vis-axis-y* (gl-objects *debug-window*)))
+      (setf (gl-objects *debug-window*)
+            (remove *vis-axis-z* (gl-objects *debug-window*))))
+    (setf length (/ length 2))
+    (let* ((object-transform
+             (cl-tf:pose->transform
```

Comment:
Please use cl-transforms or cl-transforms-stamped instead of cl-tf if you're not directly using a tf listener.